octodns/compat.py

```diff
@@ -0,0 +1,10 @@
+#
+# Python 2/3 compat bits
+#
+
+try:  # pragma: no cover
+    from StringIO import StringIO
+except ImportError:  # pragma: no cover
+    from io import StringIO
```

Comment:
Just use __six.StringIO__.

Reply:
Makes sense. I think this came out of a porting doc I was originally looking at before I moved towards using six.